pkg/scheduler/core/equivalence/eqivalence.go

```diff
@@ -126,16 +247,15 @@ func (c *Cache) RunPredicate(
 }
 
 // updateResult updates the cached result of a predicate.
-func (c *Cache) updateResult(
+// This function is thread safe for second level Cache, no need to sync with top level cache.
```

Comment:
All operations on NodeCache objects are threadsafe in the context of that NodeCache object, right? It might be worth mentioning that on the type's comment.

Reply:
Done!